app/controllers/home_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HomeController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
If this was on the application controller instead, it would make all pages need auth